features/support/helpers/helpers.js

```diff
@@ -1,9 +1,13 @@
-export const checkIfElementsInArrayAreUnique = async function (arr) {
-    arr.sort();
-     for (let i = 0; i < arr.length; i++) {
-         if (arr.indexOf(arr[i]) !== arr.lastIndexOf(arr[i])) { 
-             return false; 
-         } 
-     } 
-     return true;
- }
\ No newline at end of file
+// @flow
+
+export const checkIfElementsInArrayAreUnique = async function (
```

Comment:
this looks wrong. It doesn’t seem that it returns a Promise but rather just a boolean